Model/Fl32RuleSentRepository.php

```diff
@@ -0,0 +1,183 @@
+use Magento\Store\Model\StoreManagerInterface;
+
+class Fl32RuleSentRepository implements Fl32RuleSentRepositoryInterface
+{
+
+    protected $fl32RuleSentFactory;
+
+    protected $dataObjectHelper;
+
+    protected $extensionAttributesJoinProcessor;
+
+    private $collectionProcessor;
+
+    protected $dataFl32RuleSentFactory;
+
+    protected $searchResultsFactory;
+
+    protected $dataObjectProcessor;
+
+    protected $extensibleDataObjectConverter;
+    protected $resource;
+
+    private $storeManager;
+
+    protected $fl32RuleSentCollectionFactory;
+
+
+    /**
+     * @param ResourceFl32RuleSent $resource
+     * @param Fl32RuleSentFactory $fl32RuleSentFactory
+     * @param Fl32RuleSentInterfaceFactory $dataFl32RuleSentFactory
+     * @param Fl32RuleSentCollectionFactory $fl32RuleSentCollectionFactory
+     * @param Fl32RuleSentSearchResultsInterfaceFactory $searchResultsFactory
+     * @param DataObjectHelper $dataObjectHelper
+     * @param DataObjectProcessor $dataObjectProcessor
+     * @param StoreManagerInterface $storeManager
+     * @param CollectionProcessorInterface $collectionProcessor
+     * @param JoinProcessorInterface $extensionAttributesJoinProcessor
+     * @param ExtensibleDataObjectConverter $extensibleDataObjectConverter
+     */
+    public function __construct(
+        ResourceFl32RuleSent $resource,
+        Fl32RuleSentFactory $fl32RuleSentFactory,
+        Fl32RuleSentInterfaceFactory $dataFl32RuleSentFactory,
+        Fl32RuleSentCollectionFactory $fl32RuleSentCollectionFactory,
+        Fl32RuleSentSearchResultsInterfaceFactory $searchResultsFactory,
+        DataObjectHelper $dataObjectHelper,
+        DataObjectProcessor $dataObjectProcessor,
+        StoreManagerInterface $storeManager,
+        CollectionProcessorInterface $collectionProcessor,
+        JoinProcessorInterface $extensionAttributesJoinProcessor,
+        ExtensibleDataObjectConverter $extensibleDataObjectConverter
+    ) {
+        $this->resource = $resource;
+        $this->fl32RuleSentFactory = $fl32RuleSentFactory;
+        $this->fl32RuleSentCollectionFactory = $fl32RuleSentCollectionFactory;
+        $this->searchResultsFactory = $searchResultsFactory;
+        $this->dataObjectHelper = $dataObjectHelper;
+        $this->dataFl32RuleSentFactory = $dataFl32RuleSentFactory;
+        $this->dataObjectProcessor = $dataObjectProcessor;
+        $this->storeManager = $storeManager;
+        $this->collectionProcessor = $collectionProcessor;
+        $this->extensionAttributesJoinProcessor = $extensionAttributesJoinProcessor;
+        $this->extensibleDataObjectConverter = $extensibleDataObjectConverter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function save(
+        \Flancer32\Csp\Api\Data\Fl32RuleSentInterface $fl32RuleSent
+    ) {
+        /* if (empty($fl32RuleSent->getStoreId())) {
+            $storeId = $this->storeManager->getStore()->getId();
+            $fl32RuleSent->setStoreId($storeId);
+        } */
```

Comment:
I guess this is not needed, so it can be removed

Reply:
I think this was the way it came out of the generator. What is the reason for it to be generated?

Reply:
probably so that you can uncomment it if you have entities with store specific values (or delete if not :) )